proposals/NNNN-method-keypaths.md

```diff
@@ -15,18 +15,18 @@ expressly limited key paths to be able to reference only properties and subscrip
 \Person.pets[0]               // KeyPath<Person, Pet>
 ```
 
-This proposal adds the ability for key paths to reference methods, optionally specifying argument names:
+This proposal adds the ability for key paths to reference instance methods, optionally specifying argument names to disambiguate:
 
 ```swift
 \Person.sing                  // KeyPath<Person, () -> Sound>
-\Person.sing(melody:lyrics:)  // KeyPath<Person, (Melody,String) -> Sound>
+\Person.sing(melody:lyrics:)  // KeyPath<Person, (Melody, String) -> Sound>
 ```
 
-Note that these key paths do not provide argument values; they reference _unapplied_ methods, and the value they give is
-a closure and not the result of calling the method.
+Note that these key paths do not provide argument values: they reference _unapplied_ methods, and the value they give is
+a function type rather than the return value thereof.
```

Comment:
The value is an actual function, not a function type. But if you think the word “closure” was confusing here, we could just say “function.”

Reply:
“Function type” was a mistake, you’re right. “Function” seems to be the term favored by the [Language Guide](https://docs.swift.org/swift-book/LanguageGuide/Functions.html#ID174), at least for global functions.